addons/website/static/src/scss/website.scss

```diff
@@ -746,7 +746,16 @@ $-transition-duration: 200ms;
     display: block;
     @include o-position-absolute(0, 0, auto, 0);
     position: fixed;
-    background: $light;
+
+    // TODO remove me in master, the $light color makes no sense as is visible
+    // if the header uses rounded corners but is impossible to edit / remove.
+    // As a stable fix, it was decided to not remove it if the header uses a
+    // transparent color, in which case mixing it with $light actually alters
+    // the whole menu color.
+    $-menu-color: o-color('menu');
+    @if $-menu-color and alpha($-menu-color) < 0.95 {
```

Comment:
Isn't this missing a check in the condition to ensure the menu is not rounded?
https://watch.screencastify.com/v/o8Gs1zDdmevrHq44WyLY

Reply:
Not really no, basically, there are 4 cases:

1. The user currently has an opaque menu color & squared corners: the $light color does not change anything, we can remove it
2. The user currently has an opaque menu color & rounded courners: the $light color appears around the rounded corner, we never want that, we should remove it (base case this PR wants to solve)
3. The user currently has a transparent menu color & squared corners: the menu is not really transparent, when you are at the top of the page it is mixed with the body color (well it might let appear a body image) and when fixed, it is mixed with that $light color... it makes no sense, but we don't really want to change that in stable since it would make the fixed menu be transparent over the page content
4. The user currently has a transparent menu color & rounded corners (your video case): it's the same as 2: we don't want to see the $light color... but also the same as 3: we don't want to change that in stable. Basically, it's a case that should not exist -> the user should just not use a transparent color.

Note: for 1, we should technically not change anything in stable... but "rounded corners" is a case in our flow but there may be other custo benefiting from the fix. And, as discussed, it's really not worth the headache caring about remove that color :+1: